kubernetes/jobs/attributes-loader/cronjob.yaml

```
@@ -6,7 +6,7 @@ metadata:
   name: {{SERVICE_NAME}}
   namespace: {{NAMESPACE}}
 spec:
-  schedule: "15 6 * * *"
+  schedule: "15 9 * * *"
```

as per offline conversation
```suggestion
  schedule: "15 7 * * *"
```